lib/src/main/java/vmath/Mat3.java

```diff
@@ -0,0 +1,11 @@
+package vmath;
+
+public class Mat3 {
+    final float[] m;
+    
+    public Mat3() {
+        m = new float[]{1.0f, 0.0f, 0.0f,
+                        0.0f, 1.0f, 0.0f,
+                        0.0f, 0.0f, 1.0f};
```

Comment:
Let's go with the initialization style used [here](https://github.com/toksaitov/vmath/pull/40/files)

Reply:
Done